### PR TITLE
Fix development setup issues.

### PR DESCRIPTION
**Pull request by semtexzv**
- Instead of creating minio directories within the system directories of local machine, create them in the project directory.
- There is a port conflict between ROS backend and inventory containers
- Allow connections to kafka for both `container -> container` and `host -> container` interactions.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from ros.app import app
 
 if __name__ == "__main__":
-    app.run(host='0.0.0.0', port=8080)
+    app.run(host='0.0.0.0', port=8082)
```
Comment (patilsuraj767) [inline run.py:4]: We use the same run.py inside Dockerfile in `CMD` Also, 3scale service redirects the request to port 8080. 
I think we should consider changing the port of ingress container in docker-compose. I have raised the PR here - [https://github.com/RedHatInsights/ros-backend/pull/21/files](https://github.com/RedHatInsights/ros-backend/pull/21/files)
But this PR is a better fit for those changes.

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]
Thank for the PR @semtexzv 
Add a small concern below.

File: run.py (hunk 1 above)
Comment (semtexzv) [reply run.py:4]: Thank you, I'll either integrate changes from that PR here, or just wait for them to be merged.

Comment (patilsuraj767) [reply run.py:4]: +1 to make those changes here. As this PR is more relevant for those changes.

File: scripts/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+./minio-conf/
+./minio-data/
```
[the remark this answers is not in the document]
Comment (semtexzv) [reply scripts/.gitignore]: Thank you, fixed.

**Review by patilsuraj767 — APPROVED** [review]
Thank you @semtexzv  :+1: 
Merging the PR.

**Pull request merged**